workspaces/resource-optimization/plugins/resource-optimization-backend/README.md

```diff
@@ -1,14 +1,17 @@
-# resource-optimization
+# Resource Optimization back-end plugin
 
 Welcome to the resource-optimization backend plugin!
 
 _This plugin was created through the Backstage CLI_
 
 ## Getting started
 
-Your plugin has been added to the example app in this repository, meaning you'll be able to access it by running `yarn
-start` in the root directory, and then navigating to [/resourceOptimizationPlugin/health](http://localhost:7007/api/resourceOptimizationPlugin/health).
+This workspace includes a [devcontainer](../../.devcontainer/devcontainer.json) configuration, this is the recommended way for setting up a local dev-environment. Just open the [resource-optimization.code-workspace](../../resource-optimization.code-workspace) file with VSCode and choose to open this workspace in the preconfigured devcontiner when teh prompt appears.
```

Comment:
type 'the` instead of `teh`

Reply:
addressed in 87337da9555a9bb457d3ade3a9369ea504a6655e